app/src/main/java/de/danoeh/antennapod/fragment/HomeFragment.java

```diff
@@ -0,0 +1,402 @@
+package de.danoeh.antennapod.fragment;
```

Comment:
Never done this before, also not sure this makes sense, it's just a fragment like episodes, inbox, subscriptions and so on, so doesn't it makes sense to be in the fragments folder?

Reply:
There are more related classes, like the actual sections. Currently, if you want to change the queue screen for example, you get the fragment from one package, the adapter from another, etc. If all was just together that conceptually belongs together, it would be easier to use. In the long run, I want to get rid of the fragments folder, so I think new code should already be designed to be separated (and therefore easier to test on its own)